src/main/java/org/kumoricon/registration/order/ReprintController.java

```diff
@@ -32,8 +32,8 @@ public ReprintController(BadgePrintService badgePrintService, AttendeeRepository
         this.attendeeHistoryRepository = attendeeHistoryRepository;
     }
 
-    @RequestMapping(value = "/orders/{orderId}/attendees/{attendeeId}/reprint", method = RequestMethod.POST)
```

Comment:
not sure how bad practice it is to allow any RequestMethod mappings. if it's bad practice here i can recreate in attendee edit

Reply:
i'm just going to recreate the method, i think this probably is bad practice

Reply:
I think the POST method should be specified here, since it looks like the button that needs to invoke this endpoint is within a <form> element on the frontend and has a type of 'submit', and form submittals send a request of type POST.

Reply:
Yeah, it makes sense to move the logic from this method to editAttendeeSave, and maybe mark this one as @deprecated

Reply:
Upon taking a closer look, there is a "RePrint Badge" button when viewing an attendee that directly hits this endpoint, so adding the logic saveEditAttendee would duplicate code. I did a bit of debugging, and found that if you add the POST method back to this endpoint and then change the 'redirect:' prefix to 'forward:', both buttons will work as expected.

Reply:
nice! seems to work cleanly. 

trying to decide between the forward/redirect approach or extracting some of the reprint logic into a service-level/shared method that can be called from both OrderAttendeeEditController & ReprintController. was reading up on some threads about the different approaches and found https://stackoverflow.com/a/73111538 which seems to make sense to me.